src/sentry/static/sentry/app/components/events/interfaces/openInContextLine.jsx

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import SentryAppIcon from 'app/components/sentryAppIcon';
+import {addQueryParamsToExistingUrl} from 'app/utils/queryString';
+import styled from 'react-emotion';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+class OpenInContextLine extends React.Component {
+  static propTypes = {
+    lineNo: PropTypes.number,
+    filename: PropTypes.string,
+    components: PropTypes.array,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: false,
+    };
+  }
+
+  getUrl() {
+    const {filename, lineNo, components} = this.props;
+
+    const queryParams = {
+      lineNo,
+      filename,
+    };
+    return addQueryParamsToExistingUrl(components[0].schema.url, queryParams);
+  }
+
+  render() {
+    const {components} = this.props;
+    const url = this.getUrl();
+    return (
+      <OpenInContainer>
+        <span>{t('Open this line in:')}</span>
+        <OpenInLink data-test-id="stacktrace-link" href={url}>
+          <OpenInIcon slug={components[0].sentryApp.slug} />
+          <OpenInName>{t(`${components[0].sentryApp.name}`)}</OpenInName>
+        </OpenInLink>
+      </OpenInContainer>
+    );
+  }
+}
+
+export default OpenInContextLine;
+
+const OpenInContainer = styled('div')`
+  font-family: 'Rubik', 'Avenir Next', 'Helvetica Neue', sans-serif;
```

Comment:
This should be in `theme`?

Reply:
couldn't seem to find the theme that matched this

Reply:
`theme.text.family`